### PR TITLE
:sparkles: draw cyclejs librarie

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { div } from '@cycle/dom'
+import xs from 'xstream'
+import World from './components/world'
+import Cyclejs from './components/cyclejs'
+
+export default ({ DOM$ }) => {
+  /*
```
Comment (fabienjuif) [inline src/app.js]: Commentary is huge for nothing and it's alone.
Delete it or comment all file with `// world` style comentaries

File: src/components/cyclejs/cyclejs.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { img } from '@cycle/dom'
+import xs from 'xstream'
+import delay from 'xstream/extra/delay'
+import { WIRE_TIMEOUT } from '../../config'
+
+const stopEvent = { stop: true }
+
+const addDelay = stream$ => stream$ && stream$.compose(delay(WIRE_TIMEOUT))
```
Comment (fabienjuif) [inline src/components/cyclejs/cyclejs.js]: Maybe we have to export addDelay somewhere and use it as a library

Comment (fabienjuif) [reply src/components/cyclejs/cyclejs.js]: Same for stopEvent ?

File: src/components/cyclejs/cyclejs.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+export default ({ MUSIC$, NOTE$, HTTP$ }) => {
+  const className = '.cyclejs'
+
+  const start$ = xs.merge(
```
Comment (fabienjuif) [inline src/components/cyclejs/cyclejs.js:10]: I see that CycleJS is a Wire with a different DOM representation.
In a real life project we would merge some code.
But I bet this is better to let it be that way for the conference, this is easier to understand.

**Review by fabienjuif — CHANGES_REQUESTED** [review]
Minors commentaries

File: src/components/cyclejs/cyclejs.js (hunk 3 above)
Comment (guillaumecrespel) [reply src/components/cyclejs/cyclejs.js:10]: absolutly !

File: src/app.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { div } from '@cycle/dom'
+import xs from 'xstream'
+import World from './components/world'
+import Cyclejs from './components/cyclejs'
+import Drivers from './components/drivers'
+
+export default ({ DOM$ }) => {
+  /*
```
Comment (fabienjuif) [inline src/app.js:8]: Commentaries

**Review by fabienjuif — APPROVED** [review]

**Pull request merged**